lib/src/main/java/com/orange/ods/compose/component/list/OdsListItem.kt

```diff
@@ -377,21 +507,15 @@ private val previewParameterValues: List<OdsListItemPreviewParameter>
         val iconTypes = listOf(null, *OdsListItemIconType.values())
         val trailings: List<(@Composable () -> Unit)?> = listOf(
             null,
+            { OdsCheckboxTrailing(checked = mutableStateOf(false), enabled = true) },
```

Comment:
That would be great if we could preview the `OdsListItemTrailing` version of the API:

```kotlin
internal data class OdsListItemPreviewParameter(
    val secondaryText: String?,
    val singleLineSecondaryText: Boolean,
    val iconType: OdsListItemIconType?,
    val trailing: OdsListItemTrailing?
)
```

```kotlin
val trailings: List<OdsListItemTrailing?> = listOf(
    null,
    OdsCheckboxTrailing(checked = mutableStateOf(false), enabled = true),
    OdsSwitchTrailing(checked = mutableStateOf(false)),
    OdsIconTrailing(iconRes = android.R.drawable.ic_dialog_info, contentDescription = null),
    OdsCaptionTrailing(text = "caption")
)
```

However that would require to call both APIs in the preview method if we want to display an item without trailing element (same as `ComponentLists.kt`).